### DISCUSSION TITLE
Changing the blocktime of a running optimism network.

### DISCUSSION THREAD

**Discussion by donatopellegrino** [ethereum-optimism/developers#277] [Core Protocol] [answered]
### Did you check the documentation?

- [X] Yes

### Did you read the specs?

- [X] Yes
- [ ] No

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hi, I want to understand if it is possible to change the block time of an already running optimism stack network.
Literally:

1. I start the network;
2. The network produces some blocks, they even get finalized on L1;
3. After a while I decide I want a slower or faster blocktime.

Is this possible? If so, how? By looking at the specs, it seems that the blocktime is a constant, but it's not very clear on this specific topic. Would such a change interfere with the derivation process for example?

Thank you very much.

**Comment by opfocus** [comment]
https://docs.optimism.io/builders/chain-operators/management/configuration#blocks      
This parameter can be configured at the beginning stage of creating the OP stack.  
position：
https://github.com/ethereum-optimism/optimism/blob/a06cae81ed60c85d1566f48f45d9f9c0e48cc47c/packages/contracts-bedrock/scripts/getting-started/config.sh#L34      

I'm not sure if it will significantly affect the derivation process, potentially causing derivation failures. You can read more about the derivation process at https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/derivation.md

**Reply by donatopellegrino** [reply]
Thanks, however this doesn't answer the question.

**Comment by 1195597989** [comment, marked as answer]
Adjusting the block time of an already running Optimism network is not a standard operation and would likely require significant changes to the network’s configuration. The block time is typically defined as part of the network’s initial parameters and is not intended to be modified during operation.


The block time is set at the beginning of the document
https://docs.optimism.io/builders/node-operators/management/configuration


Once the network is running, these parameters govern its operation and changing them could disrupt the network’s stability and the finality of transactions.

**Reply by sbvegan** [reply]
Yeah this is the answer, you want a consistent block time.
